packages/dev-cli/src/commands/setup.js

```
@@ -217,6 +250,13 @@ export async function setup({ js = true, skipInstall = false }) {
         CLERK_SECRET_KEY: instance.secretKey,
         ...(js ? { NEXT_PUBLIC_CLERK_JS_URL: 'http://localhost:4000/npm/clerk.browser.js' } : {}),
       });
+
+      if (pkgJSON.scripts?.dev && pkgJSON.scripts.dev.includes('--turbo')) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)